Update issue templates (#360)
vmware/versatile-data-kit: .github/ISSUE_TEMPLATE/feature_request.md

---
name: Feature request
about: Suggest an idea for this project
title: ''
labels: enhancement
assignees: ''

---

> **Is your feature request related to a problem? Please describe.**

Reviewer: ```suggestion
**What is the feature request? What problem does it solve?**
```

> A clear and concise description of what the problem is.

Reviewer: ```suggestion
A clear and concise description of what problem needs to be solved.
```

> **Describe the solution you'd like**

Reviewer: ```suggestion
**Suggested solution**
```
